config/configuration.json

```diff
@@ -440,6 +440,15 @@
       "tags":"tags"
     }
   },
+  "BadValueTagCheck": {
```

Comment:
Could you also add this to the https://github.com/osmlab/atlas-checks/blob/dev/docs/available_checks.md doc?